connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -271,17 +256,34 @@ private static Connector instantiateConnector(Class<? extends Connector> connCla
         return result;
     }
 
-    public void stopConnector(String connName) {
+    public void stopConnectors() {
+        stopConnectors(new HashSet<>(connectors.keySet()));
+    }
+
+    public Collection<String> stopConnectors(Collection<String> connectors) {
+        final List<String> stopped = new ArrayList<>(connectors.size());
+        for (String connector: connectors) {
+            if (stopConnector(connector)) {
+                stopped.add(connector);
+            }
+        }
+        return stopped;
+    }
+
+    public boolean stopConnector(String connName) {
```

Comment:
Yeah, that's fair. Just thought I'd mention it since it stood out while reviewing the patch.